docs/2021/ui/installation.md

```diff
@@ -8,13 +8,94 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 SPDX-FileCopyrightText: 2021 Aman Dwivedi <aman.dwivedi5@gmail.com>
 SPDX-FileCopyrightText: 2021 Shruti Agarwal <mail2shruti.ag@gmail.com>
 -->
+## Requirements :scroll:
 
-1. Clone the project
-2. Install dependencies
-    - `yarn install`
-3. Start development server
-    - `yarn start`
-4. Setup CORS in FOSSology
-    - Open FOSSology and goto Admin > Customize.
-    - Lookout for "Allowed origins for REST API"
-    - Set the value to "http://localhost:3000"
+1. Your machine should have [Yarn](https://classic.yarnpkg.com/en/docs/install/#windows-stable) (preferable) or [Npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed.
+
+2. A working [FOSSology](https://github.com/fossology/fossology#installation) setup.
+
+3. Setup CORS in FOSSology 
+- Open FOSSology and go to Admin > Customize.
+- Look out for "Allowed origins for REST API"
+- Set the value to "http://localhost:3000"
+
+![image](https://user-images.githubusercontent.com/54680709/124949150-f183db80-e02e-11eb-8c5f-757a7a321d88.png)
```

Comment:
Can we use `static/img/` to host the images?

Reply:
Sure will update it

Reply:
@GMishx Updated it